src/store/store.tsx

```diff
@@ -226,40 +226,62 @@ export function Store({ children }: { children: React.ReactNode }) {
     const loadUserBalances = async (address: string) => {
         try {
             if (userBalances.length === 0) {
-                const mobulaResponse = await fetch('/api/mobula/fetch-wallet-balance', {
-                    method: 'POST',
-                    headers: {
-                        'Content-Type': 'application/json',
-                    },
-                    body: JSON.stringify({
-                        address,
-                    }),
-                })
+                const chainIds = defaultChainDetails
+                    .filter((detail) => detail.mainnet)
+                    .filter((detail) => supportedMobulaChains.map((chain) => chain.chainId).includes(detail.chainId))
+                    .map((detail) => detail.chainId)
+                    .join(',')
+
+                let attempts = 0
+                const maxAttempts = 3
+                let success = false
+
+                while (!success && attempts < maxAttempts) {
+                    try {
+                        const mobulaResponse = await fetch('/api/mobula/fetch-wallet-balance', {
+                            method: 'POST',
+                            headers: {
+                                'Content-Type': 'application/json',
+                            },
+                            body: JSON.stringify({
+                                address,
+                                chainIds,
+                                pnl: false,
+                            }),
+                        })
 
-                const mobulaResponseJson = await mobulaResponse.json()
+                        if (mobulaResponse.ok) {
+                            const mobulaResponseJson = await mobulaResponse.json()
 
-                const mappedUserBalances = mapToUserBalances(mobulaResponseJson.data).sort(
-                    (a: interfaces.IUserBalance, b: interfaces.IUserBalance) => {
-                        if (a.chainId === b.chainId) {
-                            if (a.address.toLowerCase() === '0xeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee') return -1
-                            if (b.address.toLowerCase() === '0xeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee') return 1
+                            const mappedUserBalances = mapToUserBalances(mobulaResponseJson.data).sort((a, b) => {
+                                if (a.chainId === b.chainId) {
+                                    if (a.address.toLowerCase() === '0xeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee')
```

Comment:
Is it possible that `0x00..00` address will be used for eth? Should we check for it as well?